modules/service/main.tf

```diff
@@ -794,6 +794,7 @@ data "aws_iam_policy_document" "task_exec" {
     actions = [
       "logs:CreateLogStream",
       "logs:PutLogEvents",
+      "logs:CreateLogGroup"
```

Comment:
This action is not present in `arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy`. I don't think we should accept such addition here, but I also didn't check for another solution as explained in [the AWS docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_execution_IAM_role.html).

@bryantbiggs There should be another solution. WDYT?

Reply:
I didn't find a way to inject an existing Role ARN from Inputs of the Module/Service or Module/Cluster. If you guys know some way to do that, I'll be glad.

Reply:
can you share a reproduction. The task execution role does not create log groups - this is a runtime permission which means the tasks role would be making the API call. Since the module provides the ability to create and manage the log group from Terraform, the permission is not added to the tasks role, but you can extend its permissions to add it if you wish to use the dynamic nature of creating log groups outside of IaC (just note that you won't be able to set the retention or delete them through code, it will need to be done manually)

Reply:
to solve this, I have copied the module to my internal repository and edited the configuration. I will look for in the module where to change this, to give this Input, but I didn't find exactly where to input an Iam Role for the Tasks definitions. I will cancel this Pull Request. Thank for your replies and help with it.

Reply:
again, a reproduction will go a long way. Its very difficult to even attempt to hypothesize what your configuration might look like that leads to a certain issue. I am quite confident the module supports what you are looking for today - but I don't know how I can help without seeing a reproduction so I can guide you towards a solution